public/electron.js

```diff
@@ -27,6 +27,7 @@ function createWindows() {
     y: 0,
     webPreferences: {
       nodeIntegration: true,
+      preload: __dirname + '/preload.js',
```

Comment:
I think it is better pratice to use the `path` library to join file path because paths are different on Windows and UNIX-systems and suddently the code won’t run on my Mac. You can see here https://github.com/sindresorhus/electron-boilerplate/blob/bd56521469494cbb9741cc6e930229e0dcf6ff6d/index.js#L51 for an example